.github/workflows/quay-tag.yml

```
@@ -0,0 +1,38 @@
+name: quay.tag
+run-name: Creating new tag in quay based on pushed tag in Github.
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  copy-tag-to-quay:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
+      - name: Install skopeo
+        shell: bash
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install skopeo
+      - name: Login to quay.io
+        shell: bash
+        env:
+          QUAY_TOKEN: ${{ secrets.QUAY_ROBOT_TOKEN }}
```

Update to use a secret for the quay robot username

```suggestion
          QUAY_TOKEN: ${{ secrets.QUAY_ROBOT_TOKEN }}
          QUAY_USERNAME: ${{ secrets.QUAY_ROBOT_USERNAME }}
```